src/SSA_stepper.jl

```diff
@@ -141,10 +143,25 @@ function DiffEqBase.__init(jump_prob::JumpProblem,
                                cb,_saveat,save_everystep,save_end,cur_saveat,
                                opts,tstops,1,false,true)
     cb.initialize(cb,integrator.u,prob.tspan[1],integrator)
+    DiffEqBase.initialize!(opts.callback,integrator.u,prob.tspan[1],integrator)
     integrator
 end
 
-DiffEqBase.add_tstop!(integrator::SSAIntegrator,tstop) = integrator.tstop = tstop
+function DiffEqBase.add_tstop!(integrator::SSAIntegrator{<:Any,<:Any,<:Any,<:Any,<:Any,<:Any,<:Any,<:Any,<:Vector},tstop)
```

Comment:
```suggestion
function DiffEqBase.add_tstop!(integrator::SSAIntegrator,tstop)
```

should be fine.

Reply:
Well, by default SSAIntegrator gets created with tstops=(). Thus in the default case, dynamically inserting tstops is not possible since tuples are immutable. So by default, users would get a cryptic error message like „no method insert! for Tuple{}“.

Maybe that is fine. Or it might be better to change the default tstops to be an empty array (not sure about performance consequences).

Reply:
Change it to an empty array. The tuple is likely an over-optimization.